src/core/interfaces.ts

```diff
@@ -476,6 +476,29 @@ export namespace CoreChartProps {
   export type I18nStrings = CartesianI18nStrings & PieI18nStrings;
 }
 
+export interface CoreLegendProps {
+  series: Highcharts.SeriesOptionsType[];
+  title?: string;
+  actions?: React.ReactNode;
+  position?: "bottom" | "side";
+  i18nStrings?: BaseI18nStrings;
```

Comment:
The `BaseI18nStrings` include common charts i18n, most of which do not apply to the legend. We need to have a separate CoreLegendI18nStrings type.